terraform/environments/electronic-monitoring-data/dms_iam.tf

```diff
@@ -0,0 +1,65 @@
+# Database Migration Service requires the below IAM Roles to be created before replication instances can be created. 
+
+# Define IAM role for DMS S3 Endpoint
+resource "aws_iam_role" "dms-endpoint-role" {
+  name               = "dms-endpoint-access-role-tf"
+  assume_role_policy = data.aws_iam_policy_document.dms_assume_role.json
+}
+
+# Define S3 IAM policy for DMS S3 Endpoint
+resource "aws_iam_policy" "dms-s3-ep-role-policy" {
+  name = "dms-s3-target-ep-policy"
+
+  policy = <<EOF
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Action": [
+                "s3:*",
+                "kms:*"
+            ],
+            "Resource": [
+                "arn:aws:s3::*:dms-*/*",
+                "arn:aws:s3::*:dms-*"
+            ]
+        }
+    ]
+}
```

Comment:
this will probably need to change as well - giving more specific permissions

Reply:
Hope this is ok.